runtime/proto/service/corfu_message.proto

```diff
@@ -133,6 +139,13 @@ message ResponsePayloadMsg {
     BootstrapManagementResponseMsg bootstrap_management_response = 64;
     ManagementLayoutResponseMsg management_layout_response = 65;
 
+    // LR
+    LogReplicationEntryMsg lr_entry_response = 70;
```

Comment:
Now that you've separated them, you can actually call this LogReplicationEntryACKMsg, so we easily differentiate between the replication and the ACK...

Reply:
Done.